debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SqlServerConnectorIT.java

```diff
@@ -2967,10 +2971,29 @@ private void shouldStopRetriableRestartsAtConfiguredMaximum(SqlRunnable scenario
         finally {
             // Set the database back online, since otherwise, it will be impossible to create it again
             // https://docs.microsoft.com/en-us/sql/t-sql/statements/drop-database-transact-sql?view=sql-server-ver15#general-remarks
-            connection.execute("ALTER DATABASE " + TestHelper.TEST_DATABASE_2 + " SET ONLINE");
+            try {
+                connection.execute("ALTER DATABASE " + TestHelper.TEST_DATABASE_2 + " SET ONLINE");
+            }
+            catch (SQLException e) {
+                System.out.println("Exception while setting database online " + e.getMessage());
+            }
+
+            Awaitility.await().atMost(120, TimeUnit.SECONDS)
+                    .pollInterval(5, TimeUnit.SECONDS)
+                    .until(databaseIsOnline());
         }
     }
 
+    @NotNull
+    private Callable<Boolean> databaseIsOnline() {
+
+        return () -> connection.queryAndMap("select state from sys.databases where name = '" + TEST_DATABASE_2 + "'", rs -> {
+            rs.next();
+            System.out.println("DB status " + rs.getInt(1));
```

Comment:
@mfvitale Could you please replase `Syste.out.println` in this PRs with `Testing.Print`? Thanks

Reply:
I'll address it with https://github.com/debezium/debezium/pull/5245